questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
-            "2.1",
-            "2.2.3"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure assigning internal responsibilities for open source compliance?",
-          "number": "2.g",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.3"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "오픈소스 컴플라이언스와 관련된 내부 또는 외부의 법률 전문 지식을 이용할 수 있는 방법이 확인되었습니까?",
+          "number": "2.g",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.4"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for handling review and remediation of non-compliant cases?",
-          "number": "2.h",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.4"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "오픈소스 컴플라이언스에 대한 내부 책임을 할당하는 문서화된 절차가 있습니까?",
+          "number": "2.h",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.5"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Part 3",
-      "title": "3 Open source content review and approval",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for identifying, tracking and archiving information about the open source components in a Supplied Software release?",
-          "number": "3.a",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.5"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Yes",
+      "title": "미준수 사례의 검토 및 시정을 규정하는 문서화된 절차가 있습니까?",
+      "questions": [
+        {
+          "correctAnswer": "네",
+          "question": "YES_NO",
+          "number": "3.a",
+          "type": "YES_NO",
           "specReference": [
-            "3.1",
-            "3.1.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have open source component records for the Supplied Software which demonstrate the documented procedure was properly followed?",
-          "number": "3.b",
-          "type": "YES_NO",
+            "3.1",
+            "3.1.1"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "공급 대상 소프트웨어를 구성하는 오픈소스 컴포넌트 모음에 대한 정보를 식별, 추적 및 보관하기 위한 문서화된 절차가 있습니까?",
+          "number": "3.b",
+          "type": "YES_NO",
           "specReference": [
-            "3.1",
-            "3.1.2"
-          ]
-        },
-        {
-		  "question": "Do you have a documented procedure that covers these common open source license use cases for open source components in the Supplied Software?",
-          "number": "3.c",
-          "type": "SUBQUESTIONS",
+            "3.1",
+            "3.1.2"
+          ]
+        },
+        {
+		  "question": "Yes",

```

Comment:
This does not look correct - it should be a translation of the question 3.1.2

Reply:
PR later.